packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.0.0-alpha.5",
+  "version": "6.0.0-alpha.7",
```

Comment:
Was there any need change to license package since `alpha.6`?
I don't see anything in the changelog. 🤔 
We usually don't release a package if it had not received any changes.

Reply:
I thought we were releasing a tag for package versions to be consistent?

Reply:
https://www.npmjs.com/package/@mui/x-license-pro
Nope, only changed packages most of the time.
The difference in versions between license and packages also shows this. 😉